geppetto.js/.github/workflows/main.yml

```diff
@@ -0,0 +1,25 @@
+name: Geppetto Client Workflow
```

Comment:
this will fail, among others because the working directory must be geppetto.js, we can deactivate it after the merge, I can continue with that on a different branch